apps/server/src/modules/server/admin-api-server.entity.imports.ts

```diff
@@ -0,0 +1,38 @@
+import { AccountEntity } from '@modules/account/domain/entity/account.entity';
+import { ClassEntity } from '@modules/class/entity';
+import { DeletionLogEntity, DeletionRequestEntity } from '@modules/deletion/repo/entity';
+import { GroupEntity } from '@modules/group/entity';
+import { ContextExternalToolEntity } from '@modules/tool/context-external-tool/entity';
+import { ExternalToolEntity } from '@modules/tool/external-tool/entity';
+import { SchoolExternalToolEntity } from '@modules/tool/school-external-tool/entity';
+import {
+	Course,
+	CourseGroup,
+	FederalStateEntity,
+	Role,
+	SchoolEntity,
+	SchoolYearEntity,
+	StorageProviderEntity,
+	User,
+} from '@shared/domain/entity';
+
+export const ENTITIES = [
```

Comment:
Sind hier alle benötigte Entities drin?

Reply:
Vielleicht noch mal ein Schritt weiter raus. Sollt admin-api server vielleicht ein extra module sein?

Die FileEntity war vorher drin. Ansonsten müssten wir glaube ich einmal .*Entity in jedem importieren Module suchen. Da es irgendwo dort auch eine Entity aufruf geben muss für jede Entität und in den Test auch die weiteren importieren Module abgedeckt sein sollten. 

Kann ich morgen machen, heute schaffe ich es nicht mehr.

Reply:
In dem Module wird nur UserModule und AccountModule benutzt ich denke das ist schon richtig.